cmd/websocketsource/main.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/gorilla/websocket"
+	"github.com/knative/pkg/cloudevents"
+	"github.com/pkg/errors"
+)
+
+var (
+	sink   string
+	source string
+
+	// CloudEvents specific parameters
+	eventType   string
+	eventSource string
+
+	httpClient = &http.Client{}
+)
+
+func init() {
+	flag.StringVar(&sink, "sink", "", "the host url to send messages to")
+	flag.StringVar(&source, "source", "", "the url to get messages from")
+	flag.StringVar(&eventType, "eventType", "websocket-event", "the event-type (CloudEvents)")
```

Comment:
I think we generally need some sort of "rule" for the eventTypes of our sources.

Here is what the `const`s for the other sources are:
* For instance the k8s one uses `dev.knative.k8s.event`
* GH source uses `dev.knative.source.github`
* pubsub uses `google.pubsub.topic.publish`

I think we should start w/ `dev.knative.source` ? 

/cc @vaikas-google

Reply:
oops, sorry, I totes missed this the last time around. Yes, reverse dot notation is the preferred way.